examples/deepspeed-multinode/sky.yaml

```diff
@@ -60,6 +67,10 @@ run: |
     HOSTFILE_PATH=/tmp/hostfile.${SKYPILOT_TASK_ID}
     python -c "import os;n_gpus=os.environ['SKYPILOT_NUM_GPUS_PER_NODE'];print('\n'.join([f'{ip} slots={n_gpus}' for ip in os.environ['SKYPILOT_NODE_IPS'].splitlines()]))" > ${HOSTFILE_PATH}
 
+    # Generate .deepspeed_env to propagate env vars to all workers spawned by DeepSpeed.
+    echo "Generating .deepspeed_env"
+    python3 -c 'import os; f = open(".deepspeed_env", "w"); f.write("\n".join(["{}=\"{}\"".format(var, os.getenv(var, "")) for var in os.getenv("DEEPSPEED_ENVS").split(",")])); f.write("\n"); f.close()'
```

Comment:
Can we make it multiple line to be easier to read?

Reply:
Discussed offline, decided to keep one-liner in favor of keeping the YAML short